src/account/PreferencesSubmenu.test.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import 'react-native'
+import { Provider } from 'react-redux'
+import PreferencesSubmenu from 'src/account/PreferencesSubmenu'
+import { Screens } from 'src/navigator/Screens'
+import MockedNavigator from 'test/MockedNavigator'
+import { createMockStore } from 'test/utils'
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
+import { navigate } from 'src/navigator/NavigationService'
+import { hapticFeedbackSet } from 'src/app/actions'
+
+describe('PreferencesSubmenu', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('shows the expected menu items', () => {
+    const store = createMockStore()
+    const { queryByTestId } = render(
+      <Provider store={store}>
+        <MockedNavigator component={PreferencesSubmenu}></MockedNavigator>
+      </Provider>
+    )
+    expect(queryByTestId('PreferencesSubmenu/Language')).toBeTruthy()
+    expect(queryByTestId('PreferencesSubmenu/ChangeCurrency')).toBeTruthy()
+    expect(queryByTestId('PreferencesSubmenu/HapticToggle')).toBeTruthy()
```

Comment:
same comment as before about the `getBy*` and asserting on text rather than test id where possible